SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -768,7 +779,7 @@ bool DeleteSchemaDatabase(const FString& PackagePath)
 bool GeneratedSchemaDatabaseExists()
 {
 	IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
-	
+
```

Comment:
Wow, this file had a lot of weird whitespace

Reply:
Yeah, it's really sad that Visual Studio doesn't have an option to trim trailing whitespace. I use VSCode for most of my work and have that trimming enabled which is highlighted in such PRs then